integration_test.go

```diff
@@ -33,7 +33,8 @@ const (
 	logFileName    = "logfile.log"
 	logFilePath    = "./" + logFileName
 	configFilePath = "./fluent-bit.conf"
-	numRowsData    = 10
+	numRows        = 10
+	numGoodRows    = 8
```

Comment:
Both of these constants are only used in specific tests. Better to define them as constants that are local to the scope of each of those tests.

Reply:
Done. I kept numRows a global constant as it is used in all 4 tests. I moved numGoodRows to be a local constant in the two relevant tests.